lib/scoring.rb

```diff
@@ -0,0 +1,82 @@
+#require_relative '../scrabble'  #   allows this class to access scrabble and run through irb
+
+class Scrabble::Scoring
+
+  attr_reader :word
+
+  include Scrabble
+
+  LETTER_POINTS = {
+    1 => ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2 => ["D", "G"],
+    3 => ["B", "C", "M", "P"],
+    4 => ["F", "H", "V", "W", "Y"],
+    5 => ["K"],
+    8 => ["J", "X"],
+    10 => ["Q", "Z"]
+  }
+
+  def initializer (word)
+    @word = word["word"]
+  end
+
+  # returns the score value for a single given word
+  def self.score(word)
+    score = 0
+    tiles = 0
+
+    word_array = word.upcase.split("")
+
+    word_array.each do |letter|
+      LETTER_POINTS.each do |points, letters_array|
```

Comment:
This seems to be replicating the functionality of `#char_score`, would there be a way to that method here?